test/react-native-cli/features/fixtures/rn-cli-init-android.sh

```diff
@@ -33,9 +35,8 @@ send -- $notifierVersion\r
 expect "If you want the latest version of the Bugsnag Android Gradle plugin hit enter, otherwise type the version you want"
 send -- 5.5.0-alpha01\r
 
-# TODO: Use the usual 123123... test API key
 expect "What is your Bugsnag API key?"
-send -- b161f2dbabe3204527bbe5ed4b9334a4\r
+send -- 1234567890ABCDEF1234567890ABCDEF\r
```

Comment:
There's a known issue on Android if you uses a purely numeric API key like 123123123123123..... - the entry in the manifest file gets interpreted as a Float and it falls over in a big heap.